specification/containerregistry/data-plane/Azure.ContainerRegistry/preview/2019-08-15-preview/containerregistry.json

```diff
@@ -1,3091 +1,3091 @@
-        },
-        "org.opencontainers.image.vendor": {
-          "description": "Name of the distributing entity, organization or individual.",
+          "in": "query",
+          "description": "Name of the source repository.",
+          "required": true,
+          "x-ms-parameter-location": "method"
+      },
+      "Mount": {
+          "name": "mount",
+          "description": "Digest of blob to mount from the source repository.",
           "type": "string",
-          "x-ms-client-name": "Vendor"
-        },
-        "org.opencontainers.image.licenses": {
-          "description": "License(s) under which contained software is distributed as an SPDX License Expression.",
+          "in": "query",
+          "required": true,
+          "x-ms-parameter-location": "method"
+      },
+      "Uuid": {
+          "name": "uuid",
+          "description": "A uuid identifying the upload.",
           "type": "string",
-          "x-ms-client-name": "Licenses"
-        },
-        "org.opencontainers.image.ref.name": {
-          "description": "Name of the reference for a target.",
+          "in": "path",
+          "required": true,
+          "x-ms-parameter-location": "method"
+      },
+      "Content-Range": {
+          "name": "Content-Range",
+          "in": "header",
+          "description": "Range of bytes identifying the desired block of content represented by the body. Start must the end offset retrieved via status check plus one. Note that this is a non-standard use of the `Content-Range` header.",
           "type": "string",
-          "x-ms-client-name": "Name"
-        },
-        "org.opencontainers.image.title": {
-          "description": "Human-readable title of the image",
+          "required": true,
+          "x-ms-parameter-location": "method"
+      },
+      "Range": {
+          "name": "Range",
           "type": "string",
-          "x-ms-client-name": "Title"
-        },
-        "org.opencontainers.image.description": {
-          "description": "Human-readable description of the software packaged in the image",
+          "description": "Format : bytes=<start>-<end>,  HTTP Range header specifying blob chunk.",
+          "in": "header",
+          "required": true,
+          "x-ms-parameter-location": "method"
+      },
+      "NoUploadCache": {
+          "description": "Acquired from NextLink",
+          "name": "_nouploadcache",
+          "in": "query",
+          "type": "boolean",
+          "required": false,
+          "x-ms-parameter-location": "method"
+      },
+      "State": {
+          "description": "Acquired from NextLink",
+          "name": "_state",
+          "in": "query",
           "type": "string",
-          "x-ms-client-name": "Description"
-        }
+          "required": false,
+          "x-ms-parameter-location": "method"
+      },
+      "NextLink": {
+          "name": "nextBlobUuidLink",
+          "x-ms-client-name": "nextLink",
```

Comment:
This is required for our code generator ... if you have concerns about this I can put it in a swagger transform instead.

Reply:
No problem

Reply:
How is x-ms-client-name interpreted? As per the registry API spec, the header name is "Location" for the blob upload URL, curious as to how this is  mapped to this property?